docs/source/index.rst

```diff
@@ -31,6 +31,8 @@ MineRL: Towards AI in Minecraft
   :scale: 100 %
   :alt:
 
+*These videos do not represent the new MineRL environment or the new dataset.*
```

Comment:
Here could also be a warning tag

Reply:
Tried to clean up thsi part but decided to remove it: it is bit too early in the docs to start throwing notes/warnings. They still serve as a nice visual abstract 🙃